ext/opentelemetry-exporter-cloud-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -33,6 +50,11 @@ def __init__(self, project_id=None, client=None):
         self.project_name = self.client.project_path(self.project_id)
         self._metric_descriptors = {}
         self._last_updated = {}
+        self.unique_identifier = None
+        if add_unique_identifier:
+            self.unique_identifier = "{:08x}".format(
+                random.randint(0, 16 ** 8)
```

Comment:
Just checking, this will generate different values on different instantiations after process restarts without seeding? (not very familiar with Python `random` module)